### DISCUSSION TITLE
开发者例会议题收集（2022-12-12）

### DISCUSSION THREAD

**Discussion by chuntaojun** [polarismesh/polaris#862] [Meeting]
# 开发者例会议题收集（2022-12-12）

## 会议信息

andrewshan(单家骏) 邀请您参加腾讯会议
会议主题：PolarisMesh社区双周例会
会议时间：2022/12/12 20:00-20:30 (GMT+08:00) 中国标准时间 - 北京
重复周期：2022/12/12-2023/03/21 20:00-20:30, 每两周 (周二)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/dm/bi26r1VIAx2y

#腾讯会议：825-9154-7185

## 会议记录

主题: PolarisMesh社区双周例会
日期: 2022-12-13 19:33:04
录制文件：https://meeting.tencent.com/v2/cloud-record/share?id=bb944981-0ffc-459e-85e3-83675054bfc0&from=3
访问密码：x8Sj

**Comment by atomzhong** [comment]
关于北极星开源测试体系的介绍。

**Reply by atomzhong** [reply]
[北极星开源测试体系宣讲.pptx](https://github.com/polarismesh/polaris/files/10217830/default.pptx)

**Comment by houseme** [comment]
polaris-rust 准备流程是6大步
1、注册发现 ---> v0.1.0
2、路由 ---> v0.2.0
3、限流 ---> v0.3.0
4、熔断 ---> v0.4.0
5、配置 ---> v0.5.0
6、可观测 ---> v0.6.0
欢迎大家加入一起搞哈 👏
